pyanaconda/vnc.py

```diff
@@ -1,303 +0,0 @@
-#
```

Comment:
You might want to squash this commit ("Remove vnc.py") with "Replace VNC support with GNOME remote desktop". I think they make sense together.

Reply:
Makes sense, lets do that. :)